? docs/latex/wx/manual2e.tex
? docs/latex/wx/texhelp2e.sty
Index: docs/latex/wx/cppconst.tex
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/docs/latex/wx/cppconst.tex,v
retrieving revision 1.5
diff -u -2 -r1.5 cppconst.tex
--- docs/latex/wx/cppconst.tex	2003/03/05 20:36:48	1.5
+++ docs/latex/wx/cppconst.tex	2003/06/15 15:47:48
@@ -102,5 +102,5 @@
 \twocolitem{\_\_WXDEBUG\_\_}{defined in debug mode, undefined in release mode}
 \twocolitem{wxUSE\_XXX}{if defined as $1$, feature XXX is active
-(the symbols of this form are always defined, use #if and not #ifdef to test
+(the symbols of this form are always defined, use \#if and not \#ifdef to test
  for them)}
 \end{twocollist}
Index: docs/latex/wx/db.tex
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/docs/latex/wx/db.tex,v
retrieving revision 1.30
diff -u -2 -r1.30 db.tex
--- docs/latex/wx/db.tex	2003/01/18 00:16:29	1.30
+++ docs/latex/wx/db.tex	2003/06/15 15:48:13
@@ -1422,5 +1422,5 @@
 {\it tableName} may refer to a table, view, alias or synonym.}
 \docparam{priv}{The table privilege being evaluated. May be one of the
-following (or a datasource specific privilege):
+following (or a datasource specific privilege):}
 
 \begin{verbatim}
@@ -1443,8 +1443,8 @@
                   constraint).
 \end{verbatim}
-}
+
 \docparam{userID}{{\it OPTIONAL.}  User for which to determine if the privilege
 specified to be checked is granted or not.  Default is "".
-{\it userID} is evaluated as follows:
+{\it userID} is evaluated as follows:}
 
 \begin{verbatim}
@@ -1453,9 +1453,10 @@
          userID != ""    ... UserID set equal to 'userID'
 \end{verbatim}
-}
+
 \docparam{schema}{{\it OPTIONAL.}  Owner of the table.  Specify a userID when the datasource
 you are connected to allows multiple unique tables with the same name to be
 owned by different users.  Specifying the table owner makes determination of the
-users privileges MUCH faster.  Default is NULL.  {\it userID} is evaluated as follows:
+users privileges MUCH faster.  Default is NULL.  {\it userID} is
+evaluated as follows:}
 
 \begin{verbatim}
@@ -1464,5 +1465,5 @@
          schema != ""    ... Owned by userID specified in 'schema'
 \end{verbatim}
-}
+
 \docparam{path}{{\it OPTIONAL.}  Path to the table.  Default is "".
 Currently unused.}
Index: docs/latex/wx/filesys.tex
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/docs/latex/wx/filesys.tex,v
retrieving revision 1.10
diff -u -2 -r1.10 filesys.tex
--- docs/latex/wx/filesys.tex	2003/01/18 00:16:31	1.10
+++ docs/latex/wx/filesys.tex	2003/06/15 15:48:14
@@ -126,5 +126,5 @@
 \membersection{wxFileSystem::URLToFileName}\label{wxfilesystemurltofilename}
 
-\func{static wxFileName}{URLToFileName}{\param{const wxString& }{url}}
+\func{static wxFileName}{URLToFileName}{\param{const wxString\& }{url}}
 
 Converts URL into a well-formed filename. The URL must use the {\tt file} 
Index: docs/latex/wx/frame.tex
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/docs/latex/wx/frame.tex,v
retrieving revision 1.44
diff -u -2 -r1.44 frame.tex
--- docs/latex/wx/frame.tex	2003/04/30 00:35:35	1.44
+++ docs/latex/wx/frame.tex	2003/06/15 15:48:17
@@ -524,5 +524,5 @@
 \membersection{wxFrame::SetShape}\label{wxframesetshape}
 
-\func{bool}{SetShape}{\param{const wxRegion&}{ region}}
+\func{bool}{SetShape}{\param{const wxRegion\&}{ region}}
 
 If the platform supports it, sets the shape of the window to that
Index: docs/latex/wx/log.tex
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/docs/latex/wx/log.tex,v
retrieving revision 1.25
diff -u -2 -r1.25 log.tex
--- docs/latex/wx/log.tex	2003/01/18 00:16:32	1.25
+++ docs/latex/wx/log.tex	2003/06/15 15:48:21
@@ -188,5 +188,5 @@
 \membersection{wxLog::GetTraceMasks}\label{wxloggettracemasks}
 
-\func{static const wxArrayString &}{GetTraceMasks}{\void}
+\func{static const wxArrayString \&}{GetTraceMasks}{\void}
 
 Returns the currently allowed list of string trace masks.
Index: docs/latex/wx/region.tex
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/docs/latex/wx/region.tex,v
retrieving revision 1.12
diff -u -2 -r1.12 region.tex
--- docs/latex/wx/region.tex	2003/05/03 17:54:42	1.12
+++ docs/latex/wx/region.tex	2003/06/15 15:48:24
@@ -49,6 +49,6 @@
 {\bf NB:} This constructor is only implemented for Win32 and GTK+ wxWindows ports.
 
-\func{}{wxRegion}{\param{const wxBitmap&}{ bmp},
-                  \param{const wxColour&}{ transColour = wxNullColour},
+\func{}{wxRegion}{\param{const wxBitmap\&}{ bmp},
+                  \param{const wxColour\&}{ transColour = wxNullColour},
 		  \param{int}{ tolerance = 0}}
 
@@ -184,6 +184,6 @@
 Finds the union of this region and another region.
 
-\func{bool}{Union}{\param{const wxBitmap&}{ bmp},
-                  \param{const wxColour&}{ transColour = wxNullColour},
+\func{bool}{Union}{\param{const wxBitmap\&}{ bmp},
+                  \param{const wxColour\&}{ transColour = wxNullColour},
 		  \param{int}{ tolerance = 0}}
 
Index: docs/latex/wx/scpdptr.tex
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/docs/latex/wx/scpdptr.tex,v
retrieving revision 1.3
diff -u -2 -r1.3 scpdptr.tex
--- docs/latex/wx/scpdptr.tex	2002/12/27 18:30:48	1.3
+++ docs/latex/wx/scpdptr.tex	2003/06/15 15:48:25
@@ -88,7 +88,7 @@
 crash.
 
-\membersection{wxScopedPtr::operator -\>}
+\membersection{wxScopedPtr::operator -$>$} % TODO
 
-\func{const T*}{operator -\>}{\void}
+\func{const T*}{operator -$>$}{\void} % TODO
 
 This operator works like the standard C++ pointer operator to return the pointer
Index: docs/latex/wx/statusbr.tex
===================================================================
RCS file: /pack/cvsroots/wxwindows/wxWindows/docs/latex/wx/statusbr.tex,v
retrieving revision 1.19
diff -u -2 -r1.19 statusbr.tex
--- docs/latex/wx/statusbr.tex	2003/01/18 00:16:33	1.19
+++ docs/latex/wx/statusbr.tex	2003/06/15 15:48:27
@@ -144,5 +144,5 @@
 \membersection{wxStatusBar::PushStatusText}\label{wxstatusbarpushstatustext}
 
-\func{void}{PushStatusText}{\param{const wxString&}{ string}, \param{int}{ field = 0}}
+\func{void}{PushStatusText}{\param{const wxString\&}{ string}, \param{int}{ field = 0}}
 
 Saves the current field text in a per field stack, and sets the field text

 	  	 
